dev/make_dist.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -ex
+
+FLINT_MAJOR=$1
+FLINT_MINOR=$2
+FLINT_PATCH=$3
+
+git_ref=${4-origin/flint-$FLINT_MAJOR.$FLINT_MINOR}
```

Comment:
@fredrik-johansson right now it hardcodes that `make dist` is always made from the branch `flint-3.0` (or whatever is current). I'd like to change this to instead use HEAD. This will make testing much more flexible (e.g. for GAP, we actually test making a release tarball for every pull request, that helps to catch regressions there early). Of course one does not *need* to do that.

Release then are made from a tag, and it is up to maintainers to ensure the tag is on the right commit / right branch.

Reply:
Yeah, using HEAD is definitely better.